### PR TITLE
Release 5.4.1

### PR DESCRIPTION
**Pull request by fredericDelaporte**

### CODE REVIEW

File: releasenotes.txt [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+** Bug
+
+   * #3216 Enable one-to-one optimistic lock handling in mapping
+   * #3215 Count(Distinct ...) does not work
+   * #3203 Fix a wrong example in configuration documentation
+
+** Task
+
+   * #3232 Release 5.4.1
+   * #3206 Update actions/setup-dotnet action to v3
```
Comment (bahusoid) [inline releasenotes.txt:17]: #3206 is not in 5.4.branch.

File: releasenotes.txt [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+﻿Build 5.4.1
+=============================
+
+Release notes - NHibernate - Version 5.4.1
+
+5 issues were resolved in this release.
+
+** Bug
+
+   * #3216 Enable one-to-one optimistic lock handling in mapping
```
Comment (bahusoid) [inline releasenotes.txt:10]: Need to add 5.4.0 possible breaking change note about one-to-one update leading to parent version update and the way to return back to old behavior

Comment (fredericDelaporte) [reply releasenotes.txt:10]: About docs: nhibernate/nhibernate.github.io#58.

**Review by hazzik — APPROVED** [review]

File: releasenotes.txt [hunk 3]
```diff
@@ -12,6 +33,8 @@ Release notes - NHibernate - Version 5.4.0
     * 201 issues were resolved in this release.
 
     ##### Possible Breaking Changes #####
+        * One-to-one changes does now trigger a version increment, consistently with the default behavior of other kinds of
+          associations.
```
Comment (bahusoid) [inline releasenotes.txt:37]:
```suggestion
          associations. See #3204.
```

**Pull request merged**